.changeset/giant-dolls-reply.md

```diff
@@ -0,0 +1,6 @@
+---
+'@astrojs/cloudflare': minor
+'@astrojs/vercel': minor
+---
+
+Add explicit errors when omitting output config
```

Comment:
That would cause dev/prod differences as astro dev acts differently when in static mode. Also we want adapters to support static mode in the future (they might move files around or add config files, etc), they just don't do so today.

Reply:
Yep, both great points!  Makes sense to me, thanks for clearing that up for me 👍